App/Presentation/Views/Bookmarks/EventDetails/EventDetailsBody.swift

```diff
@@ -48,19 +51,83 @@ struct EventDetailsBody: View {
             }
             DetailsBuilder(title: NSLocalizedString("Locations", comment: ""), image: "mappin.and.ellipse") {
                 if event.locations.count > 0 {
-                    ForEach(event.locations, id: \.self) { location in
-                        Text(location.locationId.capitalized)
-                            .font(.system(size: 16))
-                            .foregroundColor(.onSurface)
+                    HStack {
+                        VStack {
+                            ForEach(event.locations, id: \.self) { location in
+                                Text(location.locationId.capitalized)
+                                    .font(.system(size: 16))
+                                    .foregroundColor(.onSurface)
+                            }
+                        }
+                        Spacer()
+                        Image(systemName: "chevron.right")
+                            .foregroundColor(.onSurface.opacity(0.4))
+                            .font(.system(size: 14, weight: .semibold))
                     }
                 } else {
                     Text(NSLocalizedString("No locations listed at this time", comment: ""))
                         .font(.system(size: 16))
                         .foregroundColor(.onSurface)
                 }
             }
+            .onTapGesture {
```

Comment:
I think it might be nicer to have the sheet contain its own navigation instead of popping out another sheet. Please see [this file](https://github.com/tumble-for-kronox/Tumble-iOS/blob/main/App/Presentation/Views/Home/News/NewsItemCard.swift) for an example